docs/components.md

```diff
@@ -17,9 +17,9 @@ removed prior to the 1.0 release.
-| [otlp](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver)                                 | [metricstransform](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor)    | [signalfx](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/signalfxexporter)    | [smartagent](../internal/extension/smartagentextension)                                                                      |
+| [otlp](https://github.com/open-telemetry/opentelemetry-collector/tree/main/receiver/otlpreceiver)                                 | [metricstransform](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/processor/metricstransformprocessor)    | [signalfx](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/signalfxexporter)   | [smartagent](../internal/extension/smartagentextension)                                                                      |
 | [receiver_creator](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/receivercreator)          | [probabilisticsampler](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/probabilisticsamplerprocessor)   | [splunk_hec](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/exporter/splunkhecexporter) | [zpages](https://github.com/open-telemetry/opentelemetry-collector/tree/main/extension/zpagesextension)                      |
-| [sapm](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/sapmreceiver)                         | [resource](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/resourceprocessor)                           |                                                                                                                      |                                                                                                                              |
+| [sapm](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/receiver/sapmreceiver)                         | [resource](https://github.com/open-telemetry/opentelemetry-collector/tree/main/processor/resourceprocessor)                           |                                                                                                                      | [filestorage](https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage)     |                                                                                                                      |
```

Comment:
I believe this should be `file_storage` and it's in alpha: https://github.com/open-telemetry/opentelemetry-collector-contrib/tree/main/extension/storage/filestorage

Reply:
Thanks @rmfitzpatrick! Updated